sigs.yml

```diff
@@ -150,7 +150,7 @@
     - name: Severin Neumann
       github: svrnm
   - name: 'Semantic Conventions: LLM'
-    meeting: Every Wednesday alternating between 10:00 and 23:00 PT
+    meeting: Wednesday alternating between 10:00 PT and 14:00 UTC+8
```

Comment:
@open-telemetry/semconv-llm-approvers please take a look if I did the math for the timezones correctly